### DISCUSSION TITLE
monorepo 项目，项目 node_modules 里会有项目本身？

### DISCUSSION THREAD

**Discussion by loveyunk** [umijs/umi#7160] [Q&A]
### 🧐 问题描述

这是一个 monorepo 的项目，apps 下的项目都是使用 antd-pro 模板生成的。各项目下的 node_modules 会把项目本身也装进去。

运行 `npm run start` => `umi dev` 就会生成。

请问是什么原因导致的？

### 🚑 其他信息

![image](https://user-images.githubusercontent.com/15713020/129147816-1c6941fe-0bb5-41c2-9cf7-d93643861024.png)
![image](https://user-images.githubusercontent.com/15713020/129148163-2243a519-8cd2-4557-ad17-ff47a54032cd.png)

**Comment by GeoDaoyu** [comment]
得看看你的pnpm-workspace.yaml和pageck.json是怎么配置的。
